com.eminfedar.vaktisalah.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "com.eminfedar.vaktisalah",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "vaktisalah",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11"
+    ],
+
+    "modules": [
+        {
+            "name": "vaktisalah",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/eminfedar/vaktisalah.git",
+                    "commit": "c1534ea729f66d2073914b41b082493c6d4ff8fb"
```

Comment:
Not an actual release? We prefer actual release on Flathub.

Reply:
Ok I will make it a release :)